docs/api.yaml

```diff
@@ -0,0 +1,641 @@
+          $ref: "#/components/responses/404"
+        429:
+          $ref: "#/components/responses/429"
+        500:
+          $ref: "#/components/responses/500"
+
+  /grades:
+    post:
+      tags:
+        - "Grade"
+      summary: "Create new grade of an enrolled course"
+      description: ""
+      operationId: "createGrade"
+      requestBody:
+        description: "The grade to attribute to the student and his teaching unit"
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Grade'
+      responses:
+        200:
+          description: "The created grade"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Grade'
+        400:
+          $ref: "#/components/responses/400"
+        403:
+          $ref: "#/components/responses/403"
+        404:
+          $ref: "#/components/responses/404"
+        429:
+          $ref: "#/components/responses/429"
+        500:
+          $ref: "#/components/responses/500"
+
+  /interventions:
+    post:
+      tags:
+        - "Teacher"
+      summary: "Create new intervention hour of a teacher"
+      description: ""
+      operationId: "createIntervention"
+      requestBody:
+        description: "The interval of the intervention hours of the teacher"
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Intervention'
+      responses:
+        200:
+          description: "The created intervention"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Intervention'
+        400:
+          $ref: "#/components/responses/400"
+        403:
+          $ref: "#/components/responses/403"
+        404:
+          $ref: "#/components/responses/404"
+        429:
+          $ref: "#/components/responses/429"
+        500:
+          $ref: "#/components/responses/500"
+
+  /group:
+    patch:
+      tags:
+        - "Student"
+      summary: "Attribute a group to a student by updating his group"
+      description: ""
+      operationId: "attributeGroup"
+      requestBody:
+        description: "The student and the group to attribute him"
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Student'
+      responses:
+        200:
+          description: "The updated Student"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Student'
+        400:
+          $ref: "#/components/responses/400"
+        403:
+          $ref: "#/components/responses/403"
+        404:
+          $ref: "#/components/responses/404"
+        429:
+          $ref: "#/components/responses/429"
+        500:
+          $ref: "#/components/responses/500"
+
+  /courses/student:
```

Comment:
No. PATCH the student resource

Reply:
Adding an enrolled course to a student PATCH-ed using 'course_id' and other necessary attributes.